src/views/partials/card.handlebars

```diff
@@ -0,0 +1,52 @@
+{{#each card}}
+<div class="card-modal" hidden>
+    <button class="close-button" id="closeModal">Close Modal</button>
+    <form id="updateCard" action="/board/card/{{boardId}}/{{_id}}" method="POST">
+        <h1>{{name}}</h1>
+        <h2>Members</h2>
+        {{#each members}}
+        {{#if assigned}}
+        <h3 class="assigned">{{initials}}</h3>
```

Comment:
Not sure if this makes sense, what are initials in this case? All the initials that are assigned? Comma-separated or by space?

Reply:
For each of the members it checks if they're assigned or not, members[x].initials is their initials like JP or MP like we would display in the circle, I just imagined we would display something different for members who are assigned to the card and members who aren't yet assigned